### ISSUE TITLE
MeshLab crash on import, crash on selection...video link

### ISSUE THREAD

**Issue by ccording** [cnr-isti-vclab/meshlab#172]
MeshLab crashes when I import any STL or OBJ.

I can open these files if I right click and "Open in MeshLab".

Once opened, MeshLab crashes on choosing Filters>Cleaning and Repairing> Select Self Intersecting Faces.

Here is a video showing both crashes and workflow.
https://youtu.be/W_N8OlalIPQ

I am running macOS (Sierra 10.12.5) and MeshLab (64bit_fp_v2016.12)

Thanks for any help!
-Chris

**Comment by granzuglia** [comment]
Hi, I'm try to replicate the bug on my windows 10 system. I tried to construct a mesh like the one you have using the create filters in MeshLab and invoking some extra filters in order to get a final quad mesh similar to your own...but unfortunately (or fortunately....) i'm not able to replicate the crash. I highlight again that i'm not using Mac but Win10. Can you upload your own mesh?

Thanks a lot.

**Comment by ccording** [comment]
Hi Guido, thanks for trying it out.  You are very welcome for the video.  Here is a link to download the [cube_sphere_test.obj](https://drive.google.com/open?id=0B8_cevpYYjiuUTVISUlhcVR4RXM).

Thanks again!
-Chris

**Comment by cignoni** [comment]
I have tried on a hw/os configuration quite similar to your (Osx 10.12.5 MacBook Pro (Retina, Mid 2012)  2.6 GHz Intel Core i7 16 GB 1600 MHz DDR3 ) and it works regularly.

Is the only filter that is failing or it fails also in other cases? 
it could be connected to other things preventing the whole meshlab to works correctly on your machine...

**Comment by ccording** [comment]
Thanks for checking it out, cignoni!  MeshLab does crash on all filters for me and will not import any file that I am using whether they be .obj or .stl files.

A previous version of MeshLab did work most of the time on my current setup.  I believe it was from 2015.  I think I deleted it after clearing space on my computer.

**Comment by granzuglia** [comment]
Hi, the file you sent works also with my win10 laptop. BTW, I will not close the thread cause probably here we have a more significant problem. Can you give us some hardware specs on your laptop? Are they similar to the ones that Paolo's (cignoni) have? Can you tell me from which country are you? Sorry, if i'm asking you but sometimes we discovered problems with os localisation (for instance if the path of a file contains letters not included in the standard ascii set etc.).

**Comment by ccording** [comment]
Hi Guido.  They are similar to Paolo's specs.  Here are my specs:

MacBook Pro (Retina, 15-inch, Late 2013)
2.6 GHz Intel Core i7
16 GB 1600 MHz DDR3
NVIDIA GeForce GT 750M 2048 MB
Intel Iris Pro 1536 MB
macOS Sierra (Version 10.12.5)
Purchased and operated in USA

Thanks!
-Chris

**Comment by ccording** [comment]
Are there any steps I can take to help resolve this issue for my system?  I tried reinstalling MeshLab without success.  Can I use something to submit a stack trace?

Thanks for any help!
-Chris

**Comment by mcallieri** [comment]
Tried on another couple of machines. still unable to replicate.

**Comment by rapoell** [comment]
Same issue with the December 2016 version: crash on opening a project. It starts with some black zones on the screen, then exits.
The version 64 bit 1.3.3 still works fine (even up to 24 Gb Swap).

Config:
MacbookPro Early 2011 (AMD Radeon HD 6750M GPU, 1Gb VRAM), 8 Gb RAM DDR3
High Sierra 10.13.3

**Comment by tmhoog** [comment]
Hi, Opening *.stl and *.obj files causes Meshlab to crash on my machine. 

Config:
MBP 15" (2015), 16GB RAM DDR3, High Sierra 10.13.4.

**Comment by Ondaweb** [comment]
I'm having the same problem. Any attempted use any tool (I can select the tool, but when I have it to it's function ) causes a crash and a report to be sent to Apple. This is so for a .PLY I created, a .STL made from it, as well at the .PLY of the Stanford rabbit I downloaded from their site.  

I'm on a MacBook Air. 1.4 GHz core i5, 4GB1600Mhz DDR3 ,Intel HD Graphics 5000 1536 MB
High Sierra 10.13.6

ANY ideas appreciated.
